csharp/Ice/Secure/Client/Program.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) ZeroC, Inc.
+
+// Slice module VisitorCenter in Greeter.ice maps to C# namespace VisitorCenter.
```

Comment:
I would move this comment next to the relevant using.  Maybe on the same line?

If this is what we do in the other demos then fine to leave as is.

Reply:
Yes. The editor automatically added the other using directives and I didn't notice this was moved up.